src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/unusedimports/InputUnusedImportsInner.java

```diff
@@ -0,0 +1,18 @@
+package com.puppycrawl.tools.checkstyle.checks.imports.unusedimports;
+
+import java.util.Set;
+import java.util.List;
+import java.util.Map;
+
+import com.puppycrawl.tools.checkstyle.checks.imports.unusedimports.InputUnusedImportsInner;
+
+public class InputUnusedImportsInner {
+
+    interface List {
+        class Set {}
+    }
+
+    interface Map {}
+
+    class Set {}
```

Comment:
There are 2 Set classes, but only one import. It looks like it should be different cases (inner class and inner class in interface)

Reply:
Done